packages/ecmascript/transpilation-tests.js

```diff
@@ -62,7 +66,7 @@ class Foo {}
 class Bar extends Foo {}
 `);
 
-  test.isTrue(contains(output, 'helpers/inherits'));
+  test.isTrue(contains(output, 'helpers/builtin/inherits'));
```

Comment:
This change was necessary because of https://github.com/meteor/babel/commit/242538dea13dd4600e2ba2fc1d3bd7d242e35304, which makes [`@babel/plugin-tranform-runtime`](https://www.npmjs.com/package/@babel/plugin-transform-runtime) generate imports for `@babel/runtime/helpers/builtin/whatever` instead of `@babel/runtime/helpers/whatever`. The `helpers/builtin/*` helpers are important because they do not import any polyfills from `core-js`, and instead simply assume global APIs like `Symbol` are natively supported or have been properly polyfilled. Since Meteor takes care to provide appropriate polyfills, especially with #9439, it's important to let other parts of the framework import `core-js` polyfills, rather than leaving that job to a generic tool like Babel.